app/assets/stylesheets/pages/_tables-show.scss

```diff
@@ -1,42 +1,48 @@
 
-.top-dashboad-container {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  background-color: white;
-
-  p {
-    color: gray;
-  }
-}
 
 .top-dashboard-container-top {
+  position: sticky;
+  top: 0px;
```

Comment:
If it works then good!
But I encountered some issues with position sticky.
I had to add:
!important to your top: 0px;
and: z-index: 1 

tell me what you think